libs/blocks/merch-card-collection/merch-card-collection.js

```diff
@@ -158,6 +158,7 @@ async function initMerchCards(filtered, preferences, cardsRoot) {
   // re-order cards, update card filters
   [...cardsRoot.children].filter((card) => card.tagName === 'MERCH-CARD').forEach((merchCard) => {
     merchCard.style.display = 'none';
+    merchCard.setAttribute('daa-lh', `card-${merchCard.name || merchCard.title}`);
```

Comment:
as per https://github.com/adobecom/milo/blob/stage/libs/martech/attributes.md,

> do not add daa-lh to any element on or inside the blocks
if you add daa-ll to an element, you must have exactly 2 "levels".

I recommend using `decorateSectionAnalytics` on the cards root.

Reply:
actually we shouldn't even use this as when merch-card-collection is in a "normal" section, the quoted method *is* used (see this attribute rewritten with "normal" analytics string without catalog container) https://main--cc--adobecom.hlx.page/drafts/npeltier/promocard-test?milolibs=MWPW-148256--milo--npeltier

if you look at related ticket requirement for catalog is a much smaller string, but thinking it twice i don't see any reason setting it up in milo. Will close that PR, and raise another one specific to catalog

Reply:
do you know why we have daa-lh in your promocard PR but not at https://main--cc--adobecom.hlx.page/products/catalog?